nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -39,6 +39,8 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - [TuxClocker](https://github.com/Lurkki14/tuxclocker), a hardware control and monitoring program. Available as [programs.tuxclocker](#opt-programs.tuxclocker.enable).
 
+- [Mealie](https://nightly.mealie.io/), a self-hosted recipe manager and meal planner with a RestAPI backend and a reactive frontend application built in NuxtJS for a pleasant user experience for the whole family. Available as [services.mealie](#opt-services.mealie.enable)
+
```

Comment:
Using this change to talk about the commit in general: please just squash it to the `mealie: init at ...` one (for a cleaner history).

If you want  to keep "revisions" separate for reviewing reasons, I'd recommend looking into `git commit --fixup` and `git rebase -i --autosquash` 👍.

Reply:
I cannot squash it on top of the previous work, it was never merged before, so in `master` it's the first `mealie: init...`. Or maybe I didn't understood what you meant ? :sweat_smile:

Reply:
Sorry, to make my point clearer: IMO this PR should have four commits, in this order:

* `maintainers: add litchipi`: adds yourself to the maintainers list
* `mealie: init at <version>`: adds the package
* `nixos/mealie: init module`: add the module
* `nixos/mealie: init test`: test the moduel, and add `passthru.tests = { inherit (nixosTests) mealie; }` to the package, for CI purposes.

Usually, the documentation for a new module is added in the same commit as the module's creation, I usually add it in a separate commit at the end: do as you prefer.